tests/iana_registries/bootp_dhcp_parameters.rflx

```diff
@@ -0,0 +1,1101 @@
+       --
+       Netinfo_Tag => 113,
+
+       -- length = N, description = DHCP Captive-Portal, Ref: rfc8910
+       --
+       Dhcp_Captive_Portal => 114,
+
+       -- length = N, description = DHCP Auto-Configuration, Ref: rfc2563
+       --
+       Auto_Config => 116,
+
+       -- length = N, description = Name Service Search, Ref: rfc2937
+       --
+       Name_Service_Search => 117,
+
+       -- length = 4, description = Subnet Selection Option, Ref: rfc3011
+       --
+       Subnet_Selection_Option => 118,
+
+       -- length = N, description = DNS domain search list, Ref: rfc3397
+       --
+       Domain_Search_119 => 119,
+
+       -- length = N, description = SIP Servers DHCP Option, Ref: rfc3361
+       --
+       Sip_Servers_Dhcp_Option => 120,
+
+       -- length = N, description = Classless Static Route Option, Ref: rfc3442
+       --
+       Classless_Static_Route_Option => 121,
+
+       -- length = N, description = CableLabs Client Configuration, Ref: rfc3495
+       --
+       Ccc => 122,
+
+       -- length = 16, description = GeoConf Option, Ref: rfc6225
+       --
+       Geoconf_Option => 123,
+
+       -- description = Vendor-Identifying Vendor Class, Ref: rfc3925
+       --
+       V_I_Vendor_Class => 124,
+
+       -- description = Vendor-Identifying Vendor-Specific Information, Ref: rfc3925
+       --
+       V_I_Vendor_Specific_Information => 125,
+
+       -- Ref: rfc4578, alternative_name = Etherboot_Signature_6_Bytes_E4_45_74_68_00_00, 
+       -- alternative_name = Docsis_Full_Security_Server_Ip_Address_128, alternative_name 
+       -- = Tftp_Server_Ip_Address_For_Ip_Phone_Software_Load
+       --
+       Bootp_Vendor_Extensions_And_Dhcp_Options_128 => 128,
+
+       -- Ref: rfc4578, alternative_name = Kernel_Options_Variable_Length_String, alternat
+       -- ive_name = Call_Server_Ip_Address
+       --
+       Bootp_Vendor_Extensions_And_Dhcp_Options_129 => 129,
+
+       -- Ref: rfc4578, alternative_name = Ethernet_Interface_Variable_Length_String, alte
+       -- rnative_name = Discrimination_String_To_Identify_Vendor
+       --
+       Bootp_Vendor_Extensions_And_Dhcp_Options_130 => 130,
+
+       -- Ref: rfc4578, alternative_name = Remote_Statistics_Server_Ip_Address_131
+       --
+       Bootp_Vendor_Extensions_And_Dhcp_Options_131 => 131,
+
+       -- Ref: rfc4578, alternative_name = Ieee_802_1q_Vlan_Id
+       --
+       Bootp_Vendor_Extensions_And_Dhcp_Options_132 => 132,
+
+       -- Ref: rfc4578, alternative_name = Ieee_802_1d_P_Layer_2_Priority
+       --
+       Bootp_Vendor_Extensions_And_Dhcp_Options_133 => 133,
+
+       -- Ref: rfc4578, alternative_name = Diffserv_Code_Point_Dscp_For_Voip_Signalling_An
+       -- d_Media_Streams
+       --
+       Bootp_Vendor_Extensions_And_Dhcp_Options_134 => 134,
+
+       -- Ref: rfc4578, alternative_name = Http_Proxy_For_Phone_Specific_Applications
+       --
+       Bootp_Vendor_Extensions_And_Dhcp_Options_135 => 135,
```

Comment:
Why isn't the actual name used for these literals? There is only one "alternative_name" mentioned, so why isn't this "alternative_name" used?

Reply:
Related to https://github.com/Componolit/RecordFlux-specifications/pull/67#discussion_r655362349 and fixed in 5812afd